tools/depends/configure.ac

```diff
@@ -267,6 +267,11 @@ case $host in
           use_cpu=$host_cpu
         fi
       ;;
+      x86_64*-linux-android*)
+        if test "x$use_cpu" = "xauto"; then
+          use_cpu=$host_cpu
+        fi
+      ;;
```

Comment:
Any reason to not combine this into the above switch case? The body is exactly the same.

Reply:
Not really :smile:, thought on begin it would be better to have separated (in case different flags needed), but you are completely right. Make no sense to have separate. Becomes changed.